pkg/osquery/tables/dataflattentable/flattener-from-parser.go

```diff
@@ -0,0 +1,39 @@
+package dataflattentable
```

Comment:
any reason why this file is kebab case? would prefer snake case

Reply:
sure